dbms/src/Server/Server.cpp

```diff
@@ -261,7 +261,7 @@ struct TiFlashProxyConfig
                 args_map[engine_store_address] = config.getString("flash.service_addr");
             else
                 args_map[engine_store_advertise_address] = args_map[engine_store_address];
-            args_map[engine_label] = engine_label_value;
+            // args_map[engine_label] = engine_label_value;
```

Comment:
if there is no "flash.proxy.engine-label", then set the value default to be "tiflash" to keep the same behavior for write node.